blocks-config/image-gallery/class-spectra-image-gallery.php

```diff
@@ -1459,10 +1459,12 @@ private static function get_gallery_images( $attributes, $fetchType ) {
 		 * @param string $id       Block ID.
 		 * @param array  $attr      Array of attributes.
 		 * @param string $selector Selector to identify the carousel.
+		 * @param mixed  $lightbox_settings   Array of Lightbox Swiper Settings.
+		 * @param mixed  $thumbnail_settings  Array of Thumbnail Swiper Settings.
 		 *
 		 * @since 2.1
 		 */
```

Comment:
Can we do this in all the functions that were updated in this PR?
Also save the output_buffer in a varialbe, and return a ternary to check if is_string - if not string return empty string.
```suggestion
		 * @return string  The rendered markup or an empty string.
		 */
```

Reply:
Done